common/gcloud.py

```diff
@@ -131,3 +138,17 @@ def set_default_project(cloud_project: str):
     """Set default project for future gcloud and gsutil commands."""
     return new_process.execute(
         ['gcloud', 'config', 'set', 'project', cloud_project])
+
+
+# pylint: disable=unused-argument
+def local_create_instance(instance_name: str,
+                          instance_type: InstanceType,
+                          config: dict,
+                          metadata: dict = None,
+                          startup_script: str = None,
+                          **kwargs) -> bool:
+    """Does the equivalent of "create_instance" for local experiments, runs
+    |startup_script| in the background."""
+    command = ['/bin/bash', startup_script]
```

Comment:
i dont think /bin/bash is needed.

Reply:
Why do you think so? I'm running a script that wasn't marked executable.